docker-compose.yml

```diff
@@ -20,8 +20,8 @@ services:
       - database/username=database_user
       - database/password=database_password
     depends_on:
-      - database
-  database:
+      - frankdatabase
+  frankdatabase:
```

Comment:
Why did you change the name of the service? Will it still work with the vaue in the context.xml?

Reply:
I wanted the container names to differ from the names in file `docker-compose.test-with-openzaak.yml`. I was afraid that the containers defined by different docker-compose files would get confused.

The `docker.compose.yml` file does not work anyway. I did not investigate the link between the container names and the `context.xml` file.

Reply:
During the stand-up I asked whether to repair `docker-compose.yml`. It was more important for Marcel to fix the SoapUI tests and for me to include Ricardo's code for JWT tokens.